src/content/docs/infrastructure/host-integrations/host-integrations-list/redpanda-integration.mdx

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
The same with this one. I'd write "Redpanda dashboard" instead of "Redpanda dashboards"

Reply:
Good catch, and I agree! I'm going to make sure that's consistent in the other docs too!